moped-editor/src/utils/activityLogHelpers.js

```diff
@@ -11,7 +11,18 @@ export const formatActivityLogEntry = (change, lookupData) => {
 
   switch (change.record_type) {
     case "moped_project":
-      return formatProjectActivity(change, lookupData.entityList);
+      // if the user is changing the public process status id field,
+      // provide the publicProcessStatustList lookup object
+      if (change?.description[0]?.field === "public_process_status_id") {
+        return formatProjectActivity(
+          change,
+          lookupData.publicProcessStatustList
+        );
+      }
+      // otherwise, provide the entityList lookup object
+      else {
+        return formatProjectActivity(change, lookupData.entityList);
+      }
```

Comment:
i was on the fence about having this logic be here or in the function but I can't come up with a good reason for it not being here

Reply:
@chiaberry i went through the same thought process! i think ultimately maybe we'd want to name the `lookupData` objects the same thing as the table names and then we would just be able to interpolate that into the logic of the component? that would be a departure from our naming schema though. it didn't feel necessary to do that for this one case but i imagine there will be more and i could tackle it then